src/apps/vpn/ui/screens/settings/ViewPrivacy.qml

```diff
@@ -53,6 +54,12 @@ VPNViewBase {
                         Accessible.role: Accessible.StaticText
                         Accessible.name: text
                     }
+                    Connections {
+                        target: VPNController
+                        function onStateChanged() {
+                            disconnectWarningLoader.Layout.preferredHeight = disconnectWarningLoader.active ? disconnectWarningLoader.implicitHeight : 0;
+                        }
+                    }
```

Comment:
Hmmm, maybe putting this text block in a loader was a bad suggestion by me since this is kind of ugly

We can se `visible: active` on the loader, or just do away with the loader completely here. One day I'd like to profile if the former is actually any better